samples/components/dialects/Hive12Dialect.tdd

```diff
@@ -1196,10 +1238,130 @@
+    <aggregation value='AGG_DAY' />
+    <aggregation value='AGG_HOUR' />
+    <aggregation value='AGG_MINUTE' />
+    <aggregation value='AGG_SECOND' />
+    <aggregation value='AGG_WEEK' />
+    <aggregation value='AGG_MONTHYEAR' />
+    <aggregation value='AGG_MDY' />
+    <aggregation value='TRUNC_YEAR' />
+    <aggregation value='TRUNC_QTR' />
+    <aggregation value='TRUNC_MONTH' />
+    <aggregation value='TRUNC_WEEK' />
+    <aggregation value='TRUNC_DAY' />
+    <aggregation value='TRUNC_HOUR' />
+    <aggregation value='TRUNC_MINUTE' />
+    <aggregation value='TRUNC_SECOND' />
+  </supported-aggregations>
+
+  <sql-format>
+    <format-column-definition>
+      <local-type name='int'>
+        <remote-type name='I1' value='TINYINT' />
+        <remote-type name='I2' value='SMALLINT' />
+        <remote-type name='I4' value='INT' />
+        <remote-type name='default' value='BIGINT' />
+      </local-type>
+      <local-type name='real'>
+        <remote-type name='R4' value='FLOAT' />
+        <remote-type name='default' value='DOUBLE' />
+      </local-type>
+      <local-type name='str'>
+        <remote-type name='default' value='STRING' />
+      </local-type>
+      <local-type name='datetime'>
+        <remote-type name='default' value='TIMESTAMP' />
+      </local-type>
+      <local-type name='date'>
+        <remote-type name='default' value='DATE' />
+      </local-type>
+      <local-type name='bool'>
+        <remote-type name='default' value='BOOLEAN' />
+      </local-type>
+    </format-column-definition>
+    <format-date-literal formula="CAST('%1' AS DATE)"  format='yyyy-MM-dd' />
+    <format-datetime-literal formula="CAST('%1' AS TIMESTAMP)" format='yyyy-MM-dd HH:mm:ss' />
+    <format-is-distinct value='Operator' />
+    <format-null>
+      <local-type name='bool' value='CAST(NULL AS BOOLEAN)' />
+      <local-type name='int' value='CAST(NULL AS INT)' />
+      <local-type name='real' value='CAST(NULL AS DOUBLE)' />
+      <local-type name='str' value='CAST(NULL AS STRING)' />
+      <local-type name='date' value='CAST(NULL AS DATE)' />
+      <local-type name='datetime' value='CAST(NULL AS TIMESTAMP)' />
+    </format-null>
+    <format-select>
+      <part name='Select' value='SELECT %1' />
+      <part name='Into' value='INTO %1' />
+      <part name='From' value='FROM %1' />
+      <part name='Where' value='WHERE %1' />
+      <part name='Group' value='GROUP BY %1' />
+      <part name='Having' value='HAVING %1' />
+      <part name='OrderBy' value='ORDER BY %1' />
+      <part name='Top' value='LIMIT %1' />
+      <part name='TopSampleRecords' value='ORDER BY rand()\nLIMIT %1' />
+    </format-select>
+    <format-simple-case value='BalancedIIF' />
+    <format-true value='true' />
+    <id-allowed-characters value='_abcdefghijklmnopqrstuvwxyz0123456789' />
+    <id-case value='Lower' />
+    <id-max-length value='63' />
+    <id-quotes value='`' />
+    <start-of-week-offset value='0' />
```

Comment:
```suggestion
```
Based on your other comments we might as well remove this, since it isn't accurate